packages/regl-worldview/src/commands/GLText.js

```diff
@@ -188,28 +206,41 @@ const frag = `
   uniform mat4 projection;
   uniform sampler2D atlas;
   uniform float cutoff;
+  uniform bool scaleInvariant;
+  uniform float scaleInvariantSize;
 
   varying vec2 vTexCoord;
   varying float vEnableBackground;
   varying vec4 vForegroundColor;
   varying vec4 vBackgroundColor;
   varying vec4 vHighlightColor;
   varying float vEnableHighlight;
+  varying float vBillboard;
+
   void main() {
     float dist = texture2D(atlas, vTexCoord).a;
 
     // fwidth(dist) is used to provide some anti-aliasing. However it's currently only used
     // when the solid background is enabled, because the alpha blending and
     // depth test don't work together nicely for partially-transparent pixels.
     float edgeStep = smoothstep(1.0 - cutoff - fwidth(dist), 1.0 - cutoff, dist);
+
+    if (scaleInvariant && vBillboard == 1.0 && scaleInvariantSize < 0.03) {
```

Comment:
Just curious if you have run into any precision issues with `vBillboard == 1.0`? I recall that while working on this originally I had some strange experiences with the `varying` interpolation producing values that were very close but not exactly 1.0, so just want to confirm that you aren't experiencing this.

Reply:
I'm not seeing precision issues. I prefer to use boolean values for flags (or avoid flags completely), but I didn't want to introduce more changes than needed.

Reply:
Yes, I think I ran into an issue where regl / WebGL 1 don't support `uniform bool` in fragment shaders (or something like that)